community/stats/index.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: single
+title:  "Usage Statistics"
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+## Why is data being collected?
+
```

Comment:
Maybe a sentence here, without a heading.

"[If enabled], Spinnaker collects data about how the tool is being used. This data is anonymized and aggregated.

Or words to that effect. "If enabled" would link to the thing you have below about opt-in/opt-out.

Reply:
Done.